internal/services/systemcentervirtualmachinemanager/system_center_virtual_machine_manager_virtual_machine_instance_resource_test.go

```diff
@@ -0,0 +1,370 @@
+	}
+
+	return utils.Bool(resp.Model != nil), nil
+}
+
+func (r SystemCenterVirtualMachineManagerVirtualMachineInstanceResource) basic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_system_center_virtual_machine_manager_virtual_machine_instance" "test" {
+  scoped_resource_id = azurerm_arc_machine.test.id
+  custom_location_id = azurerm_system_center_virtual_machine_manager_server.test.custom_location_id
+
+  infrastructure {
+    checkpoint_type                                                 = "Standard"
+    system_center_virtual_machine_manager_cloud_id                  = azurerm_system_center_virtual_machine_manager_cloud.test.id
+    system_center_virtual_machine_manager_template_id               = azurerm_system_center_virtual_machine_manager_virtual_machine_template.test.id
+    system_center_virtual_machine_manager_virtual_machine_server_id = azurerm_system_center_virtual_machine_manager_server.test.id
+  }
+
+  operating_system {
+    computer_name = "testComputer"
+  }
+
+  hardware {
+    cpu_count    = 1
+    memory_in_mb = 1024
+  }
+
+  network_interface {
+    name              = "testNIC"
+    ipv4_address_type = "Dynamic"
+    ipv6_address_type = "Dynamic"
+    mac_address_type  = "Dynamic"
+  }
+
+  lifecycle {
+    // Service API always provisions a virtual disk with bus type IDE per Virtual Machine Template by default
+    ignore_changes = [storage_disk]
+  }
+}
+`, r.template(data))
+}
+
+func (r SystemCenterVirtualMachineManagerVirtualMachineInstanceResource) requiresImport(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_system_center_virtual_machine_manager_virtual_machine_instance" "import" {
+  scoped_resource_id = azurerm_system_center_virtual_machine_manager_virtual_machine_instance.test.scoped_resource_id
+  custom_location_id = azurerm_system_center_virtual_machine_manager_virtual_machine_instance.test.custom_location_id
+
+  infrastructure {
+    checkpoint_type                                                 = "Standard"
+    system_center_virtual_machine_manager_cloud_id                  = azurerm_system_center_virtual_machine_manager_cloud.test.id
+    system_center_virtual_machine_manager_template_id               = azurerm_system_center_virtual_machine_manager_virtual_machine_template.test.id
+    system_center_virtual_machine_manager_virtual_machine_server_id = azurerm_system_center_virtual_machine_manager_server.test.id
+  }
+
+  operating_system {
+    computer_name = "testComputer"
+  }
+
+  hardware {
+    cpu_count    = 1
+    memory_in_mb = 1024
+  }
+
+  network_interface {
+    name              = "testNIC"
+    ipv4_address_type = "Dynamic"
+    ipv6_address_type = "Dynamic"
+    mac_address_type  = "Dynamic"
+  }
+}
+`, r.basic(data))
+}
+
+func (r SystemCenterVirtualMachineManagerVirtualMachineInstanceResource) complete(data acceptance.TestData) string {
```

Comment:
`system_center_virtual_machine_manager_inventory_item_id`, `virtual_network_id`, `storage_disk` are not test ?

Reply:
For `storage_disk`, I have added it to test case.

As `system_center_virtual_machine_manager_inventory_item_id` requires the existing System Center Virtual Machine Manager Server with System Center Virtual Machine Instance in the host machine and the test custom location provided by service team only allows one System Center Virtual Machine Manager Server. So we can't get two System Center Virtual Machine Manager Servers at the same time in the test environment. So I didn't add it to test case.

As the test host machine doesn't support for `virtual_network_id`, so I didn't add to test case.